subsys/bluetooth/services/Kconfig.tps

```diff
@@ -0,0 +1,15 @@
+# Bluetooth GATT Tx Power service
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BT_GATT_TPS
```

Comment:
I think we should make it BT_GATT_TPS_SRV so that we are prepared for a client in the future.

Reply:
Yes, sure, I added "_srv" suffix API and file names